doc/source/users_guide/cime-change-namelist.rst

```diff
@@ -50,11 +55,140 @@ On the hand, to change the driver namelist value of the starting year/month/day,
 Note that
 
 To see the result of change, call `preview_namelists <../Tools_user/preview_namelists.html>`_  and verify that the new value appears in **CaseDocs/drv_in**.
+
+.. _basic_example:
+
+Setting up a multi-year run
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This shows all of the steps necessary to do a multi-year simulation starting from a "cold start" for all components.  The
+compset and resolution in this example are for CESM but the steps are similar for other compets.
+
+1. Create a new case named EXAMPLE_CASE in your **$HOME** directory.
+
+   ::
+
+      > cd $CIME/scripts
+      > ./create_newcase --case ~/EXAMPLE_CASE --compset B1850 --res f09_g17
+
+2. Check the pe-layout by running **./pelayout**. Make sure it is suitable for your machine.
+   If it is not use `xmlchange <../Tools_user/xmlchange.html>`_ or  `pelayout <../Tools_user/pelayout.html>`_ to modify your pe-layout.
+   Then setup your case and build your executable.
+
+   ::
+
+      > cd ~/EXAMPLE_CASE
+      > ./case.setup
+      > ./case.build
+
+   .. warning:: The case.build script is can be compute intensive and may not be suitable to run on a login node. As an alternative you would submit this job to an interactive queue.
+                For example, on the NCAR cheyenne platform, you would use **qcmd -- ./case.build** to do this.
+
+3. In your case directory, set the job to run 12 model months, set the wallclock time, and submit the job.
+
+   ::
+
+      > ./xmlchange STOP_OPTION=nmonths
+      > ./xmlchange STOP_N=12
+      > ./xmlchange JOB_WALLCLOCK_TIME=06:00 --subgroup case.run
+      > ./case.submit
+
+4. Make sure the run succeeded.
+
+   You should see the following line or similar at the end of the **cpl.log** file in your run directory or your short term archiving directory, set by ``$DOUT_S_ROOT``.
+
+   ::
+
+      (seq_mct_drv): ===============       SUCCESSFUL TERMINATION OF CPL7-cesm ===============
+
+5. In the same case directory, Set the case to resubmit itself 10 times so it will run a total of 11 years (including the initial year), and resubmit the case. (Note that a resubmit will automatically change the run to be a continuation run).
+
+   ::
+
+      > ./xmlchange RESUBMIT=10
+      > ./case.submit
+
+   By default resubmitted runs are not submitted until the previous run is completed.  For 10 1-year runs as configured in this
+   example, CIME will first submit a job for one year, then when its finished submit a job for another year.  There will be
```

Comment:
change "then when its finished" to "when that job completes it will"

Reply:
ok